__tests__/app.test.js

```diff
@@ -50,6 +40,24 @@ describe('/api/topics', () => {
   })
 });
 
+describe.only('/api/articles', () => {
```

Comment:
all `.only` and `.skip` syntax should be removed from your tests prior to merging your branch into main, this is so we can be sure that all the tests are passing before code is pushed to main